substrate/frame/support/src/traits/tokens/fungible/imbalance.rs

```diff
@@ -43,7 +43,6 @@ impl<Balance> HandleImbalanceDrop<Balance> for () {
 ///
 /// Importantly, it has a special `Drop` impl, and cannot be created outside of this module.
 #[must_use]
-#[derive(RuntimeDebug, Eq, PartialEq)]
```

Comment:
there is a separate PR for this part https://github.com/paritytech/polkadot-sdk/pull/1717